backend/src/main/java/ca/bc/gov/hlth/mohums/util/FilterUserByOrgId.java

```diff
@@ -0,0 +1,89 @@
+package ca.bc.gov.hlth.mohums.util;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.function.Predicate;
+import net.minidev.json.JSONObject;
+import net.minidev.json.parser.JSONParser;
+import net.minidev.json.parser.ParseException;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.expression.EvaluationContext;
+import org.springframework.expression.Expression;
+import org.springframework.expression.ExpressionException;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.expression.spel.support.StandardEvaluationContext;
+
+/**
+ *
+ */
+public final class FilterUserByOrgId implements Predicate<Object> {
+
+    private static final int ABBREVIATED_USER_ENTITY_LENGTH = 64;
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(AuthorizedClientsParser.class);
+
+    private final String orgId;
+    private final JSONParser jsonParser;
+    private final Expression expression;
+
+    public FilterUserByOrgId(final String id) {
+        this.orgId = id;
+        this.jsonParser = new JSONParser(JSONParser.DEFAULT_PERMISSIVE_MODE);
```

Comment:
I don't think JSONParser is thread-safe. The class doesn't say whether it is, which usually means we assume it's not. Also, I found this in the author's GitHub:

https://github.com/netplex/json-smart-v2/blob/master/json-smart/src/main/java/net/minidev/json/parser/JSONParserString.java#L23

To avoid issues, make it a local variable in `extractOrgId`. (Then that method can also become `private static`.)